cockatrice/src/game/filters/filter_string.cpp

```diff
@@ -10,15 +10,15 @@
 peg::parser search(R"(
 Start <- QueryPartList
 ~ws <- [ ]+
-QueryPartList <- ComplexQueryPart ( ws ("and" ws)? ComplexQueryPart)* ws*
+QueryPartList <- ComplexQueryPart ( ws ("AND" ws)? ComplexQueryPart)* ws*
 
-ComplexQueryPart <- SomewhatComplexQueryPart ws $or<[oO][rR]> ws ComplexQueryPart / SomewhatComplexQueryPart
+ComplexQueryPart <- SomewhatComplexQueryPart ws $or<[O][R]> ws ComplexQueryPart / SomewhatComplexQueryPart
```

Comment:
considering removing the lowercase letters makes this a group of one, writing it without the brackets has the same effect
```suggestion
ComplexQueryPart <- SomewhatComplexQueryPart ws $or<OR> ws ComplexQueryPart / SomewhatComplexQueryPart
```

Reply:
I've tried doing that and it resulted in it crashing but I've made a change that should essentially be the same.